### DISCUSSION TITLE
这个诗词库玩笑开大了吧，其实并不全

### DISCUSSION THREAD

**Discussion by LanxingYizhi** [chinese-poetry/chinese-poetry#267] [Show and tell]
前边写了个软件，信心满满的和朋友表示上边有最全的古诗词，结果好友考了句陶渊明的诗：“采菊东篱下，悠然见南山。
山气日夕佳，飞鸟相与还。”
结果直接没有结果，后来又考了几首陶渊明，同样没有结果
尴尬死了

**Comment by jackeyGao** [comment]
在 #268 中已有说明。 三国和魏晋阶段的诗词的确很少， 前几个月有同学提供了曹操诗集， 后面会继续补充合并。 

但没有开玩笑， 的确是最全。

**Comment by Deali-Axy** [comment]
确实
